Dynatrace/dynatrace-operator
# Update `CONTAINERS_COUNT` environment variable if sidecar container was added by another mutating webhook

# Description
Currently `CONTAINERS_COUNT` environment variable is set once when initCointainer with OneAgent installer is created.
If there are more mutating webhooks, e.g. istio, additional sidecar containers are not taken into account.

This change adds support for such scenario by setting correct `COINTAINERS_COUNT` in webook reinvoke phase.

## How can this be tested?
1. setup dybakube with applicationMonitoring
2. install istio
3. create two namespaces, one istio-injected, one not
4. deploy pod in both namespaces and check if `CONTAINERS_COUNT` is set to 2 and 1 respectively


## Checklist
